sql/pg_duckdb--0.2.0--0.3.0.sql

```diff
@@ -17,3 +17,227 @@ CREATE AGGREGATE @extschema@.approx_count_distinct(anyelement)
 
 CREATE DOMAIN pg_catalog.blob AS bytea;
 COMMENT ON DOMAIN pg_catalog.blob IS 'The DuckDB BLOB alias for BYTEA';
+
+-- json_exists
+CREATE FUNCTION @extschema@.json_exists(json VARCHAR , path VARCHAR)
```

Comment:
It seems confusing that these functions expect a `VARCHAR` type for the `json` argument. I'd expect that argument to be of the `json` or `jsonb` (or have functions for both of them).